docs/library/spi_engine/instruction-format.rst

```diff
@@ -197,9 +203,17 @@ bus behavior.
    * - Bits
      - Name
      - Description
-   * - [7:3]
+   * - [7:4]
      - reserved
      - Must always be 0.
+   * - [3]
+     - cs_active_high
+     - Configures the polarity of CS. When 0 (default), CS is active-low as 
+     - usual. If set to 1, CS will be active-high. Note that there's no need 
+     - to change the commands: for cs_active_high=1, the value of CS will be 
+     - automatically inverted with respect to the command. This was introduced
+     - in version 1.02.71 of the core, setting this bit on earlier versions has
+     - no effect.
```

Comment:
Long paragraphs are not prefixed by `-`
```suggestion
     - Configures the polarity of CS. When 0 (default), CS is active-low as 
       usual. If set to 1, CS is active-high. Note that the command instructions
       remain unchanged because the value of CS is inverted at the output
       register and additional logic occurs when the CS active state is asserted
       (e.g., reset counters).
       This was introduced in version 1.02.71 of the core, setting this bit on
       earlier versions has no effect.
```
Rephrased to unhide reset states at `CMD_CHIPSELECT` and use imperative grammar.